### DISCUSSION TITLE
How to get rid of the red-colored underline in a signature help?

### DISCUSSION THREAD

**Discussion by creasty** [neoclide/coc.nvim#4053] [Q&A] [answered]
Where is this color coming from?

In the README:

<img width="1167" alt="Screen Shot 2022-08-13 at 12 06 41" src="https://user-images.githubusercontent.com/1695538/184466225-39a440e5-c6a5-4908-b048-c92f0f3744e1.png">

In my nvim:

<img width="701" alt="Screen Shot 2022-08-13 at 12 10 23" src="https://user-images.githubusercontent.com/1695538/184466222-71fa86ae-3fc5-483a-86e7-114cd9ddfc2d.png">

<img width="859" alt="Screen Shot 2022-08-13 at 12 09 45" src="https://user-images.githubusercontent.com/1695538/184466233-7698d457-0c45-45c0-9583-bb53c1d53a9b.png">

**Comment by chemzqm** [comment, marked as answer]
Don't know why, should be issue with nvim on some terminals, vim doesn't have red underline.
